ArchiSteamFarm/Web/Responses/HtmlDocumentResponse.cs

```diff
@@ -23,37 +23,32 @@
 using System.Threading.Tasks;
 using AngleSharp;
 using AngleSharp.Dom;
-using ArchiSteamFarm.Core;
 using JetBrains.Annotations;
 
 namespace ArchiSteamFarm.Web.Responses;
 
 public sealed class HtmlDocumentResponse : BasicResponse, IDisposable {
```

Comment:
I'd prefer not to, those are two completely different classes doing something else. Object response is specifically for JSON-related deserialization, and not HTML parsing.

Reply:
In my mind the `ObjectResponse<T>` always could contain any object no matter how it ways deserialized, because there is no comment or other documentation on it being JSON specific. Maybe we can document this for the future?